jest.config.js

```diff
@@ -3,6 +3,6 @@ module.exports = {
   preset: 'ts-jest',
   runner: '@jest-runner/electron/main',
   testEnvironment: 'node',
-  testRegex: '(/tests/.*|(\\.|/)(test|spec))\\.ts$',
+  testMatch: ['**/?(*.)+(spec|test).ts'],
```

Comment:
update, to be able to run tests locally for a file or just 1 test

Reply:
`yarn test ./tests/some.spec.ts` was working well previously, but this seems working too :)

Reply:
jest nodeConfig - will run 1 file 
jest -t 'name of the test' will run only 1 specific test 

to simplify the test creation process.